tests/test_intermediary.py

```diff
@@ -74,7 +74,6 @@ def checkResponseFields(self, response):
         self.assertEqual(response.upstream, 'github')
         self.assertEqual(response.comments, [{'body': 'mock_body', 'name': 'mock_name', 'author': 'mock_author',
                                               'changed': None, 'date_created': 'mock_date', 'id': 'mock_id'}])
-        self.assertEqual(response.priority, None)
```

Comment:
Should this assertion be replaced with something else?  (Is simply deleting it the correct thing to do?)

Reply:
This assertion was part of `checkResponseFields(...)` function which includes common assertions.  This worked until now as the value for `priority` was indeed `None`, because there some previous attempt to implement the priority field syncing. This PR introduce `priority` syncing, so common assertion will not work anymore.